utils/rowgroup/rowaggregation.cpp

```diff
@@ -1405,6 +1408,18 @@ void RowAggregation::doSum(const Row& rowIn, int64_t colIn, int64_t colOut, int
         {
             fRow.setBinaryField_offset(dec, sizeof(*dec), offset);
         }
+    }
+    else
+    {
+        if (LIKELY(!isNull(fRowGroupOut, fRow, colOut)))
+        {
+            long double valOut = fRow.getLongDoubleField(colOut);
+            fRow.setLongDoubleField(valIn+valOut, colOut);
+        }
+        else
+        {
+            fRow.setLongDoubleField(valIn, colOut);
+        }
     } // end-of isWideDataType block
```

Comment:
This comment becomes misguiding after the change.

Reply:
++